src/Miso/Canvas.hs

```diff
@@ -51,18 +51,18 @@
 --      secs = secs' + fromIntegral n
 --      millis = millis' + fromIntegral n
 --    globalCompositeOperation DestinationOver
---    clearRect (0,0,300,300)
+--    clearRect (0, 0, 300, 300)
 --    fillStyle $ Canvas.color (rgba 0 0 0 0.6)
 --    strokeStyle $ Canvas.color (rgba 0 153 255 0.4)
 --    save ()
 --    translate (150, 150)
---    rotate ((((2 * pi) / 60) * secs) + (((2 * pi) / 60000) * millis))
+--    rotate ((((2 * pi) \/ 60) * secs) + (((2 * pi) \/ 60000) * millis))
```

Comment:
Note that / in haddock is markup for italics and needs to be escaped if it's to be rendered literally.